modules/oe_multilingual_front_page/README.md

```diff
@@ -0,0 +1,7 @@
+# OpenEuropa Multilingual Front page
+
+The module ensures that "index" is set as homepage alias.
```

Comment:
The module ensures that the homepage alias is alsways maintained when linking to the front page.